lib/addons/karpenter/index.ts

```diff
@@ -202,6 +210,7 @@ export class KarpenterAddOn extends HelmAddOn {
                         subnetSelector: subnetTags,
                         securityGroupSelector: sgTags,
                     },
+                    ttlSecondsUntilExpired: ttlSecondsUntilExpired,
```

Comment:
did you validate that null setting does not break the install? I saw cases when null not permitted, however lack of key is OK.

Reply:
Yes - just validated and it works as intended (no value provided -> null -> no custom helm value provided)